src/components/AvatarWrapper/AvatarWrapper.vue

```diff
@@ -174,10 +178,6 @@ export default {
 					return !this.failed ? '' : 'icon-user'
 				case ATTENDEE.ACTOR_TYPE.FEDERATED_USERS:
 					return (this.token && !this.failed) ? '' : 'icon-user'
-				case ATTENDEE.ACTOR_TYPE.EMAILS:
-					return this.token === 'new' ? 'icon-mail' : (this.hasCustomName ? '' : 'icon-user')
-				case ATTENDEE.ACTOR_TYPE.GUESTS:
-					return this.hasCustomName ? '' : 'icon-user'
```

Comment:
Works fine for me, if you put 'guest' and 'bot' in the empty strings instead of making `characterIconClass` computed